Source/MLX/Documentation.docc/MLX.md

```diff
@@ -79,10 +79,11 @@ are the CPU and GPU.
 - ``HasDType``
 - ``ScalarOrArray``
 
-### Argument Types
+### Parameter Types
 
 - ``IntOrPair``
 - ``IntOrArray``
+- ``FloatOrArray``
```

Comment:
This is new and will be used in the new layers.  Not used in this commit but I got it set up so am including it.